README.md

```diff
@@ -79,7 +80,21 @@ Ease Probe supports the following probing methods:
         contain: easeprobe
   ```
 
-- **Client**. Currently, support the following native client. Support the mTLS. ( [Native Client Probe](#35-native-client-probe) )
+- **Host**. Run a SSH command on remote host and check the CPU, Memory, and Disk usage. ( [Host Load Probe](#35-host-resource-usage-probe-configuration) )
+
+  ```yaml
+  host:
+    servers:
+      - name : server
+        host: ubuntu@172.20.2.202:22
+        key: /path/to/server.pem
+        threshold:
+          cpu: 0.80  # cpu usage  80%
+          mem: 0.70  # memory usage 70%
+          disk: 0.90  # disk usage 90%
+  ```
```

Comment:
Need to add a description to explain the relationship between thresholds condition. From my understanding, it's the `AND` relationship. Notification will be sent only when all meters are greater than the threshold.

Reply:
I think the `OR` conditions would make more sense. I will state that more clearly